config/foreman.migrations/20231005004305_redis_default_cache.rb

```diff
@@ -0,0 +1,3 @@
+if answers['foreman'].is_a?(Hash) && answers['foreman']['rails_cache_store'] == { 'type' => 'file' }
```

Comment:
Would it be better to make this more robust and only check if type equals file?
```suggestion
if answers['foreman'].is_a?(Hash) && answers['foreman']['rails_cache_store'].is_a?(Hash) && answers['foreman']['rails_cache_store']['type'] == 'file'
```